crates/flo/src/builders.rs

```diff
@@ -863,6 +864,74 @@ impl<'a> BlockBuilder<'a> {
     ) -> VariableId {
         self.transmute_into_new_variable(source_var, target_type, vec![], None)
     }
+
+    /// Adds a statement that relocates the provided `block` identifier into the
+    /// specified `target_var`.
+    ///
+    /// # Arguments
+    ///
+    /// * `target_var` - The SSA variable of type `ptr` into which the `block`
+    ///   identifier will be relocated.
+    /// * `block` - The identifier of the block to relocate.
+    pub fn get_block_address(
+        &mut self,
+        target_var: VariableId,
+        block: BlockId,
+        diagnostics: Vec<DiagnosticId>,
+        location: Option<LocationId>,
+    ) -> StatementId {
+        let statement = GetBlockAddressStatement {
+            target_var,
+            block,
+            diagnostics,
+            location,
+        };
+
+        self.add_statement(&Statement::GetBlockAddress(statement))
+    }
+
+    /// Adds a statement that relocates the provided `block` identifier into a
+    /// new variable.
+    ///
+    /// # Arguments
+    ///
+    /// * `block` - The identifier of the block to relocate.
+    pub fn get_block_address_into_new_variable(
```

Comment:
For consistency with e.g. the constants, can this be `get_new_block_address`? (And same for the simple variant.)